corehq/apps/data_interfaces/static/data_interfaces/js/case_dedupe_main.js

```diff
@@ -247,7 +247,8 @@ hqDefine("data_interfaces/js/case_dedupe_main", [
             initialPageData.get('case_properties'),
             initialPageData.get('include_closed'),
             initialPageData.get('properties_to_update'),
-            initialPageData.get('all_case_properties')
+            initialPageData.get('all_case_properties'),
+            initialPageData.get('allow_actions')
```

Comment:
maybe i'm missing something, but I'm not seeing the `initial_page_data` for allow actions? I don't see it even when I view the entire file here. How is that being populated?

Reply:
Also it looks like the [signature for CaseDeduple](https://github.com/dimagi/commcare-hq/blob/ce73a3b88c1cecfc94c2304d665e88cde6cddcaa/corehq/apps/data_interfaces/static/data_interfaces/js/case_dedupe_main.js#L157-L166) doesn't appear to have `allowActions` as a used parameter. Is this line still being used?

Reply:
Thanks! This looks like a line I forgot to remove from a previous iteration. Updated in 088e875c8a00cf7d97d20ed36c03d7b267b20474